src/components/SignIn.js

```diff
@@ -189,15 +160,13 @@ export default function SignIn() {
         </Box>
       </Box>
       {/* useLogin hook errors */}
-      {errors && (
-        <Snackbar
-          open={open}
-          autoHideDuration={6000}
-          onClose={handleClose}
-          message={errors.msg} // Use the error message from the errors object
-          action={action}
-        />
-      )}
+      <Snackbar
+        open={open}
```

Comment:
Option 2 for opening this on an error is to just make the Snackbar.open prop be looking at whether the `errors` variable from the hook is populated or not:

```
<Snackbar
  open={!!errors}
  autoHideDuration={6000}
  onClose={handleClose}
  message={errors?.msg} // Use the error message from the errors object
  action={action}
 />
```

Reply:
> Option 2 for opening this on an error is to just make the Snackbar.open prop be looking at whether the `errors` variable from the hook is populated or not:
> 
> ```
> <Snackbar
>   open={!!errors}
>   autoHideDuration={6000}
>   onClose={handleClose}
>   message={errors?.msg} // Use the error message from the errors object
>   action={action}
>  />
> ```

Sorry, I did this before, but it did undone and I forgot to update this part too.